reference/docs-conceptual/learn/ps101/06-flow-control.md

```diff
@@ -0,0 +1,336 @@
+---
+title: Flow control | PowerShell 101
+description: PowerShell provides methods to create loops, make decisions, and logically control the flow of code in scripts.
+ms.date: 06/02/2020
+ms.topic: guide
+ms.custom: Contributor-mikefrobbins
+ms.reviewer: mirobb
+---
+# Chapter 6 - Flow control
+
+## Scripting
+
+When you move from writing PowerShell one-liners to writing scripts, it sounds a lot more
+complicated than it really is. A script is nothing more than the same or similar commands that you
+would run interactively in the PowerShell console, except they're saved as a `.PS1` file. There are
+some scripting constructs that you may use such as a `foreach` loop instead of the `ForEach-Object`
+cmdlet. To beginners, the differences can be confusing especially when you consider that `foreach`
+is both a scripting construct and an alias for the `ForEach-Object` cmdlet.
+
+## Looping
+
+One of the great things about PowerShell is, once you figure out how to do something for one item,
+it's almost as easy to do the same task for hundreds of items. Simply loop through the items using
+one of the many different types of loops in PowerShell.
+
+### ForEach-Object
+
+`ForEach-Object` is a cmdlet for iterating through items in a pipeline such as with PowerShell
+one-liners. `ForEach-Object` streams the objects through the pipeline.
+
+Although the Module parameter of `Get-Command` accepts multiple values that are strings, it only
```

Comment:
Should "Module" be bold since it's a parameter?  This is in a few spots.

Reply:
Fixed this and a couple others.